plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -513,7 +513,7 @@
         "memory": {
             "arguments": {
                 "intercept": 0,
```

Comment:
I was tempted to add in a few words for extra overhead here but thought better of it.